on build, `$service-worker` exports `assets` instead of `files`

**Describe the bug**
According to docs, the $service-worker module should export the variable `files`, an array of URL strings representing the files in your static directory. This variable to exported as `assets` instead

**Logs**
Please include browser console and server logs around the time this bug occurred.
<details>

```

> Executing task: npm run build <


> my-app@0.0.1 build /Users/alamode/my-app
> svelte-kit build

vite v2.1.2 building for production...
✓ 17 modules transformed.
.svelte/output/client/_app/manifest.json                            0.67kb
.svelte/output/client/_app/assets/start-a8cd1609.css                0.16kb / brotli: 0.10kb
.svelte/output/client/_app/assets/pages/index.svelte-ec482201.css   0.81kb / brotli: 0.34kb
.svelte/output/client/_app/pages/index.svelte-7163c483.js           1.61kb / brotli: 0.69kb
.svelte/output/client/_app/chunks/index-21894282.js                 4.69kb / brotli: 1.85kb
.svelte/output/client/_app/start-499c645e.js                        15.38kb / brotli: 5.35kb
vite v2.1.2 building SSR bundle for production...
✓ 14 modules transformed.
.svelte/output/server/app.js      68.20kb
.svelte/output/server/style.css   1.00kb
vite v2.1.2 building for production...
✓ 2 modules transformed.
'files' is not exported by .svelte/build/runtime/service-worker.js, imported by src/service-worker.js
file: /Users/alamode/my-app/src/service-worker.js:1:16
1: import { build, files, timestamp } from "$service-worker";
                   ^
2: 
3: //--  uncomment 2 lines below, and comment line above
> 'files' is not exported by .svelte/build/runtime/service-worker.js, imported by src/service-worker.js
Error: 'files' is not exported by .svelte/build/runtime/service-worker.js, imported by src/service-worker.js
    at error (/Users/alamode/my-app/node_modules/rollup/dist/shared/rollup.js:5308:30)
    at Module.error (/Users/alamode/my-app/node_modules/rollup/dist/shared/rollup.js:9720:16)
    at Module.traceVariable (/Users/alamode/my-app/node_modules/rollup/dist/shared/rollup.js:10108:29)
    at ModuleScope.findVariable (/Users/alamode/my-app/node_modules/rollup/dist/shared/rollup.js:8875:39)
    at Identifier.bind (/Users/alamode/my-app/node_modules/rollup/dist/shared/rollup.js:4043:40)
    at CallExpression.bind (/Users/alamode/my-app/node_modules/rollup/dist/shared/rollup.js:2741:31)
    at CallExpression.bind (/Users/alamode/my-app/node_modules/rollup/dist/shared/rollup.js:6755:15)
    at ExpressionStatement.bind (/Users/alamode/my-app/node_modules/rollup/dist/shared/rollup.js:2745:23)
    at Program.bind (/Users/alamode/my-app/node_modules/rollup/dist/shared/rollup.js:2741:31)
    at Module.bindReferences (/Users/alamode/my-app/node_modules/rollup/dist/shared/rollup.js:9716:18)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! my-app@0.0.1 build: `svelte-kit build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the my-app@0.0.1 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/alamode/.npm/_logs/2021-03-24T18_33_33_546Z-debug.log
The terminal process "/bin/bash '-c', 'npm run build'" terminated with exit code: 1.

```
</details>

**To Reproduce**
Repro is here: https://github.com/iltimasd/kit-sw-files-bug

It was generated using `npm init svelte@next`, and then adding the `src/service-worker.js`

```js
 //-- src/service-worker.js`
 import { build, files, timestamp } from "$service-worker";

//--  uncomment 2 lines below, and comment line above
// import { build, assets, timestamp } from "$service-worker";
// const files = assets;

console.log(files);

```

**Expected behavior**
A clear and concise description of what you expected to happen.

According to Docs and Migration guide the `$service-worker` should export `files`.



**updated env as i previously ran this in the wrong dir

  System:
    OS: macOS Mojave 10.14.6
    CPU: (8) x64 Intel(R) Core(TM) i7-8557U CPU @ 1.70GHz
    Memory: 797.71 MB / 16.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 14.15.1 - /usr/local/bin/node
    Yarn: 1.22.4 - /usr/local/bin/yarn
    npm: 6.14.8 - /usr/local/bin/npm
  Browsers:
    Chrome: 89.0.4389.90
    Edge: 89.0.774.57
    Firefox: 84.0.2
    Safari: 13.0.4
  npmPackages:
    @sveltejs/kit: next => 1.0.0-next.58 
    svelte: ^3.29.0 => 3.35.0 

**Severity**
How severe an issue is this bug to you? Is this annoying, blocking some users, blocking an upgrade or blocking your usage of SvelteKit entirely?

Note: the more honest and specific you are here the more we will take you seriously.

It is an easy fix, however the docs could making it confusing as to why a build would fail.

--- a/.changeset/sharp-days-exercise.md
+++ b/.changeset/sharp-days-exercise.md
@@ -1,0 +1,5 @@
+---
+'@sveltejs/kit': patch
+---
+
+Rename \$service-worker::assets to files, per the docs

--- a/packages/kit/src/core/build/index.js
+++ b/packages/kit/src/core/build/index.js
@@ -471,7 +471,7 @@ async function build_service_worker(
 					.join(',\n\t\t\t\t')}
 			];
 
-			export const assets = [
+			export const files = [
 				${manifest.assets
 					.map((asset) => `${s(`${config.kit.paths.base}/${asset.file}`)}`)
 					.join(',\n\t\t\t\t')}

